src/meta/buildTags.tsx

```diff
@@ -605,12 +605,12 @@ const buildTags = (config: BuildTagsParams) => {
       );
     }
 
-    if (config.openGraph.site_name) {
+    if (config.openGraph.siteName) {
       tagsToRender.push(
         <meta
           key="og:site_name"
           property="og:site_name"
-          content={config.openGraph.site_name}
+          content={config.openGraph.siteName}
```

Comment:
```suggestion
          content={config.openGraph.siteName || config.openGraph.site_name}
```

This will allow this change to be backwards compatible.

Reply:
Hello thanks, I take into consideration the changes 🙏